.github/workflows/release.yml

```diff
@@ -0,0 +1,237 @@
+# This workflow handles the complete release process for static-code-analysis
+# It converts snapshot to release version, deploys to Maven Central via OSSRH,
+# and then bumps to the next snapshot version
+
+name: Release to Maven Central
+permissions:
+  contents: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version (e.g., 0.18.0)'
+        required: true
+        type: string
+      next_snapshot_version:
+        description: 'Next snapshot version (e.g., 0.19.0-SNAPSHOT)'
+        required: true
+        type: string
+
+jobs:
+  release:
+    name: Release to Maven Central
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
```

Comment:
```suggestion
      uses: actions/checkout@v6
```

Reply:
latest is [v6](https://api.github.com/repos/actions/checkout/releases/latest), should work here as well...